### PR TITLE
Update database from our tool_destinations.yml

### PR DESCRIPTION
**Pull request by mira-miracoli**

### CODE REVIEW

File: tools.yml [hunk 1]
```diff
@@ -1019,6 +1023,13 @@ tools:
     mem: 10
   toolshed.g2.bx.psu.edu/repos/iuc/hypo/hypo/.*:
     cores: 8
+  toolshed.g2.bx.psu.edu/repos/bgruening/instagraal/instagraal/.*:
+    cores: 1
+    gpus: 1
+    mem: 30
+    scheduling:
+      accept:
```
Comment (bgruening) [inline tools.yml:1031]: This tools __needs__ a GPU. Should this then be `required`?

**Comment by bgruening** [comment]
I guess you need to run `tpv format` on this file.

**Comment by bgruening** [comment]
@cat-bro could you please also have a look. Thanks.

File: tools.yml (hunk 1 above)
Comment (bgruening) [reply tools.yml]: Could be, I hope @cat-bro or @nuwang can give some advice here.

Comment (nuwang) [reply tools.yml]: Specifying gpus should be sufficient.

**Comment by bgruening** [comment]
Just added:

```
disco:
  cores: 8
  env:
    DISCO_MAX_MEMORY: 24
```

Sorry.

**Comment by cat-bro** [comment]
The gpu tag isn't necessary: gpus: 1 or however many should take care of mapping to a destination with the right GPU count. Two things about disco: If this is the toolshed tool https://toolshed.g2.bx.psu.edu/view/iuc/disco/dfefaf2bab95 then it is more consistent to use the full id. There is an env var setting max memory but mem is not set at all. Shouldn't mem be set to the same value?

**Comment by cat-bro** [comment]
Otherwise this looks good to me

File: tools.yml [hunk 2]
```diff
@@ -952,6 +952,11 @@ tools:
     cores: 8
   toolshed.g2.bx.psu.edu/repos/iuc/dexseq/dexseq_count/.*:
     mem: 25
+  toolshed.g2.bx.psu.edu/repos/iuc/disco/disco/.*:
+    cores: 8
+    mem: 24
+    env:
+      DISCO_MAX_MEMORY: 24
```
Comment (bgruening) [inline tools.yml:959]:
```suggestion
      DISCO_MAX_MEMORY: {mem}
```

**Comment by mira-miracoli** [comment]
yes you are right, thank you @nuwang

**Comment by bgruening** [comment]
lint is failing, but otherwise looks good to me.

**Comment by mira-miracoli** [comment]
image

This is why the lint fails. so we should ignore it, I guess

File: tools.yml [hunk 3]
```diff
@@ -911,18 +945,27 @@ tools:
...
     mem: 8
+  toolshed.g2.bx.psu.edu/repos/iuc/dexseq/dexseq/.*:
+    cores: 8
   toolshed.g2.bx.psu.edu/repos/iuc/dexseq/dexseq_count/.*:
     mem: 25
+  toolshed.g2.bx.psu.edu/repos/iuc/disco/disco/.*:
+    cores: 8
+    mem: 24
+    env:
+      DISCO_MAX_MEMORY: {mem}
```
Comment (nuwang) [inline tools.yml:956]:
```suggestion
      DISCO_MAX_MEMORY: "{mem}"
```
Needs double quotes, or the yaml parser will think it's a dict

File: tools.yml [hunk 4]
```diff
@@ -911,18 +945,27 @@ tools:
...
     mem: 8
+  toolshed.g2.bx.psu.edu/repos/iuc/dexseq/dexseq/.*:
+    cores: 8
   toolshed.g2.bx.psu.edu/repos/iuc/dexseq/dexseq_count/.*:
     mem: 25
+  toolshed.g2.bx.psu.edu/repos/iuc/disco/disco/.*:
+    cores: 8
+    mem: 24
+    env:
+      DISCO_MAX_MEMORY: "{mem}"
```
Comment (nuwang) [inline tools.yml:956]: Looks like it has to be single quotes or tpv format will complain
```suggestion
      DISCO_MAX_MEMORY: '{mem}'
```

**Comment by bgruening** [comment]
I tried a rebase ... hope I did not messed it up.

**Pull request merged**